services/portal/coreMetadataPage/coreMetadataPageQuestions.js

```diff
@@ -0,0 +1,57 @@
+const chai = require('chai');
+const coreMetadataPageProps = require('./coreMetadataPageProps.js');
+
+const I = actor();
+
+// helper functions that come from portal
+const typeTransform = function (type) {
+  let t = type.replace(/_/g, ' '); // '-' to ' '
+  t = t.replace(/\b\w/g, l => l.toUpperCase()); // capitalize words
+  return `| ${t} |`;
+};
+
+const fileSizeTransform = function (fileSize) {
+  const i = fileSize === 0 ? 0 : Math.floor(Math.log(fileSize) / Math.log(1024));
+  const fileSizeStr = (fileSize / (1024 ** i)).toFixed(2) * 1;
+  const suffix = ['B', 'KB', 'MB', 'GB', 'TB'][i];
+  return `${fileSizeStr} ${suffix}`;
+};
+
+const dateTransform = date => `Updated on ${date.substr(0, 10)}`;
+
+/**
+ * coreMetadataPage Questions
+ */
+module.exports = {
+  doesCoreMetadataPageLooksCorrect(metadata) {
+    console.log('validate the appearance of core metadata page for a given file');
+    I.seeElement(coreMetadataPageProps.coreMetadataPageBox1Class);
+    I.seeElement(coreMetadataPageProps.coreMetadataPageBox2Class);
+    I.seeElement(coreMetadataPageProps.coreMetadataPageBox3Class);
+
+    chai.expect(metadata).to.not.be.undefined;
+
+    for (const metadataKey of Object.keys(metadata)) {
+      switch (metadataKey) {
+        // no show for 'project_id'
+        case 'project_id':
+          I.dontSee(metadata[metadataKey]);
+          break;
+        // special cases for 'file_size', 'type' and 'updated_datetime'
+        case 'file_size':
+          I.see(fileSizeTransform(metadata[metadataKey]));
```

Comment:
i'm not sure copying the transform functions from data-portal is the way to go 🤔 
i would ignore project_id, type, file_size and updated_datetime (and keep the comments that explain why) and just look for other fields as well as "more data info" and "download" button
this is just a suggestion though :)

Reply:
a ha! good call, I should check for the button and table title